sources/threads.c

```diff
@@ -4769,6 +4769,76 @@ int IniSortBlocks(int numworkers)
 
 /*
   	#] IniSortBlocks : 
+  	#[ UpdateSortBlocks :
+*/
+ 
+/**
+ *	A version of IniSortBlocks which only updates the pointers in the master's
+ *	buffer, to be used after reallocation of that buffer.
+ */
+
```

Comment:
Doxygen is happy with blank lines between a function and its corresponding comment (but #579 for `threads.c`), while VSCode doesn't seem to like them. Personally, I also prefer not to put blank lines between them, as it looks tidier.

Reply:
Many of the functions in threads.c have a blank line, though. We can remove them all, if you prefer?

Reply:
Yes, but removing all such blank lines (maybe also in other files) would be handled in a separate PR. For now, we can focus on the newly added function.